.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+<!--
+This issue template is meant mainly for bug reports and feature requests.
+
+If you are looking for help, you can reach out to the IRC channel
+#heketi on freenode or use the mailing list heketi-devel@gluster.org.
+
+You can still use an issue to ask for help. In this case
+you should remove the template contents or use it as a guideline
+for providing information for debugging.
+-->
+
+### Kind of issue
+
+> Uncomment only one of these:
+> 
+> Bug
+> Feature request
+
+### Observed behavior
+
+
+### Expected/desired behavior
+
+
+### Details on how to reproduce (minimal and precise)
+
+
+### Other useful information?
+
+
+### Information about the enviroment:
+
+- Heketi version used (e.g. v6.0.0 or master): 
+- Operating system used: 
+- Heketi compiled from sources, as a package (rpm/deb), or container: 
+- If container, which container image: 
+- Using kubernetes, openshift, or direct install: 
+- If kubernetes/openshif, gluster running in kube/opebshift or outside: 
+- More information: 
```

Comment:
We need to also probably ask for deployment method. Custom deployment config or gk-deploy.

Reply:
right, i'll add that (and fix the openshift typos...) ;-)